renderer/src/components/Plausible.tsx

```diff
@@ -0,0 +1,17 @@
+import { Helmet, HelmetProvider } from 'react-helmet-async'
+
+const Plausible = () => {
+  return (
+    <HelmetProvider>
+      <Helmet>
```

Comment:
I am not familiar with Helmet, but I find it a bit surprising that we are using both `HelmetProvide` and `Helmet` elements here. Shouldn't we put `<HelmetProvider>` to `App.tsx` to make `Helmet` available to all pages and components?

Reply:
good point!